src/js/structs/UniversePackagesVersions.js

```diff
@@ -0,0 +1,27 @@
+import Item from "./Item";
+
+const sortPackageVersions = selectedPackage => {
+  if (selectedPackage.packageVersions == null) {
+    return [];
+  }
+  const versions = selectedPackage.packageVersions;
+
+  return Object.keys(versions).sort((a, b) => {
+    if (+versions[a] > +versions[b]) {
+      return -1;
+    }
+    if (+versions[a] < +versions[b]) {
+      return 1;
+    }
+
+    return 0;
+  });
+};
+
+class UniversePackagesVersions extends Item {
+  getPackageByName(packageName) {
```

Comment:
The thing that this method is "getting" is an sorted array of package versions, but it's named like it's getting a a package. I think something like `getVersionsForPackage()` is much better.

Reply:
Was a design decision to have the sort by default.

I believe `getPackageByName` is more clear than `getVersionsForPackage` since that's what it's doing, but I'm happy to change if that's a blocker for you.

Reply:
Talked offline: will change to `getPackageVersionsByName`